bin/node/rpc/src/lib.rs

```diff
@@ -95,12 +100,12 @@ pub struct FullDeps<C, P, SC> {
 }
 
 /// A IO handler that uses all Full RPC extensions.
-pub type IoHandler = jsonrpc_core::IoHandler<sc_rpc::Metadata>;
+pub type IoHandler = jsonrpc_core::MetaIoHandler<sc_rpc::Metadata>;
```

Comment:
Change to `MetaIoHandler` is not necessary. `IoHandler<sc_rpc::Metadata>` is pretty much the same thing, the former allows you to configure middlewares, but since you are not doing that anyway, you can stick to the simpler `IoHandler`.